.github/actions/setup-node/action.yaml

```
@@ -0,0 +1,14 @@
+  This action sets up Node.js for use in actions by taking advantage of the
+  official Node.js Docker image. It also caches dependencies to speed up
+  subsequent runs.
+runs:
+  using: composite
+  steps:
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '22'
+        cache: 'npm'
+        cache-dependency-path: apps/web/package-lock.json
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)